WordPress/WordPressApi/WordPressComApi.m

```diff
@@ -7,7 +7,7 @@
 #import "NotificationsManager.h"
 #import "WPUserAgent.h"
 
-static NSString *const WordPressComApiClientEndpointURL = @"https://public-api.wordpress.com/rest/v1.1/";
+static NSString *const WordPressComApiClientEndpointURL = @"https://public-api.wordpress.com/rest/";
```

Comment:
Well, I can definitely add those assertions, since this would be pretty fresh, and they'd help us debug this feature as well.  Good recommendation.

The dev should notice the problems when testing the service, though (since it'd fail).  But that's just a side-note... the assertions would be a nice addition.